installers/install.sh

```diff
@@ -419,3 +397,11 @@ else
     manual_update_instructions
   fi
 fi
+
+if [ -n "${ACTIVATE}" ]; then
+  # switch this shell to interactive mode
+  set -i
+  # control flow of this script ends with this line: replace the shell with the activated project's shell
+  exec $STATEPATH activate ${ACTIVATE}
+fi
+info "You may now start using the '$STATEEXE' program."
```

Comment:
Are you developing on Windows? If not, how did the lack of newline happen?!

Reply:
I marked approved thinking of this as a nit, but on Linux/Darwin, the newline is kinda/sorta required.